torch/fx/symbolic_trace.py

```diff
@@ -188,6 +188,38 @@ def forward(*args, **kwargs):
             torch.nn.Module.__call__ = orig_call
         return self.graph
 
+def wrap(orig_function):
+    """
+    A decorator that can be put on existing functions to use them in fx.
+
+        @torch.fx.wrap
+        def my_custom_function(x, y):
+            return x * x + y * y
+
+        sqrt = torch.fx.wrap(math.sqrt)
+
+    When called during symbolic tracing, the function will be inserted in the
+    graph rather than tracing it. When called outside of tracing,
+    the function will execute normally.
+    """
+
+    def wrapped(*args, **kwargs):
```

Comment:
need @functools.wraps to maintain same name for function?
Also, we could add support for symtrace function (as an optional argument to wrap).

Reply:
Added `wraps` in the new PR. On the subject of `__symtrace__`, I think we should have a discussion over the holidays about upstreaming `__symtrace__` into main FX, since it's not there yet